source/manage-apps/deploy/deployment-models-and-regions.txt

```diff
@@ -32,65 +32,175 @@ Deployment Models
 Global Deployment
 ~~~~~~~~~~~~~~~~~
 
-Globally deployed App Services Apps are hosted in every :ref:`cloud
-region <realm-regions>` that App Services supports. The a Realm SDKs
-automatically execute application requests, such as reading user
-information or calling a function in the cloud region that's closest to
-the user that issued the request.
-
-Write operations to a linked MongoDB cluster are handled within a single
-region that you specify when you :ref:`create your application
-<create-a-realm-app>`. By default, App Services handles writes in
-the region closest to your application's original linked MongoDB
-cluster.
+A globally deployed app is available in multiple regions across the
+world. Any global region can receive and process incoming requests for a
+global app.
+
+All write operations to a linked data source are handled within a single
+region that you specify when you create the app. You can choose any
+global region to handle writes, but should generally choose the region
+closest to your MongoDB cluster.
+
+Global apps are currently deployed to the following regions:
+
+- Ireland (IE) - ``aws-eu-west-1``
+- Oregon (US-OR) - ``aws-us-west-2``
+- Sydney (AU) - ``aws-ap-southeast-2``
+- Virginia (US-VA) - ``aws-us-east-1``
+
+.. note::
+   
+   Realm SDKs automatically execute application requests, such as
+   reading user information or calling a function, in the global
+   deployment region that's closest to the user that issued the request.
```

Comment:
i think this probably should be higher up in the text and not a note since it's important info. would make sense when explaining how global apps process requests. perhaps after

>  Any global region can receive and process incoming requests for a
global app.

Reply:
also would be good to discuss how a global app routes requests from clients not using the Realm SDKs. 

for example
- how does a global app route data api or graphql requests?
- where are functions invoked by triggers executed?
- anything about rules or schema validation worth noting?